kuma/urls_beta.py

```diff
@@ -50,6 +53,16 @@
         name='attachments.mindtouch_file_redirect'),
 ]
 
+# Add the signin and signout urls, and profile-related urls
+# TODO: These lines also pull in a number of endpoints that we don't
+# need on the beta site, so we should prune these down when the beta
+# site becomes the default.
+urlpatterns += [url('users/', include('kuma.users.urls'))]
+urlpatterns += i18n_patterns(url('',
+                                 decorator_include(never_cache,
+                                                   users_lang_urlpatterns)))
```

Comment:
@davidflanagan As you say in your comment, I'd like to prune these down to what we actually need for handling logins/logouts. I'll look more into that next week.

Reply:
After a quick look, I think we could prune the auth-key endpoints, the account recover endpoints, and the profile endpoints perhaps as well?

Reply:
The profile endpoints were a nice bonus, but they're not necessary and we could cut them.

As a practical matter, I find django's urlpatterns/include() system of registering routes impenetrable, and find myself incapable of doing anything other than copy and paste. And also kuma.users.urls seems like a particularly complex one, so I think I could use some pointers about how to enable just a subset of the users urls...

Reply:
And so much of it is changing in Django 2. Sigh. 

Would it help to revisit this a bit "deeper". Perhaps I've never understood it fully but currently we use a piece of middleware to dynamically switch the urlconf stuff around, in runtime, based on the domain used. Perhaps, if urlconf is clunky, we revisit that approach and instead of having "multiple urlconf" we have other tricks to swap between wiki and beta. Not sure what but perhaps we can reevaluate now that the beta project has come a longer way and is, at least mentally, more mature.

Reply:
So I solved my problem by just creating another urls_beta.py file in kuma/users/ and modifying this file to use that one instead of the original non-beta urls.